site/app/controllers/PollController.php

```diff
@@ -129,7 +129,7 @@ public function addNewPoll() {
                 new RedirectResponse($this->core->buildCourseUrl(['polls']))
             );
         }
-        $date = \DateTime::createFromFormat("Y-m-d", $_POST["release_date"]);
+        $date = \DateTime::createFromFormat("m/d/Y", $_POST["release_date"]);
```

Comment:
We can pull a date config from the config so it works internationally and so we have the option to change all the formatting in 1 place in the future if needed.

E.g:
```php
$this->core->getConfig()->getDateTimeFormat()->getFormat('gradeable')
``` 
You can see a list of the different date formats in the file `site/app/models/DateTimeFormat.php` and choose whichever works, `gradeable` is a common one that is used in the site, if nothing works you can add a specific format for 'polls' and use that instead.

Reply:
are you saying I should replace  $date = \DateTime::createFromFormat("m/d/Y", $_POST["release_date"]); with 
$date = \DateTime::createFromFormat($this->core->getConfig()->getDateTimeFormat()->getFormat('gradeable'), $_POST["release_date"]); ?

Reply:
Yeah to clarify `$this->core->getConfig()->getDateTimeFormat()->getFormat('gradeable')` returns a date format (its type is a string) so you can use that wherever you have a hardcoded value, or pass it from PHP to twig. If the gradeable format doesn't work for you, you can pick a different one or create one specific for polls.

Reply:
Ok thanks I didn't realize that, I'll go back and make some changes.